applications/plot_simtel_histograms.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python3
+
+'''
+    Summary
+    -------
+    This application prints sim_telarray histograms into pdf file.
```

Comment:
Just to make it clear (cause reading it I was a little confused), I suggest to change to `plots and prints`.

Reply:
Done